doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1627,6 +1779,12 @@ Numeral notations
             As noted above, the :n:`(abstract after @num)` directive has no
             effect when :n:`@qualid__parse` lands in an :g:`option` type.
 
+         .. exn:: 'via' and 'abstract' cannot be used together.
+
+            Since the parser is not actually reduced with :n:`abstract`,
+            its result cannot be translated from :n:`@qualid__ind` to
+            the actual type.
```

Comment:
I don't understand what you mean by "is not actually reduced with".

Reply:
When you pass `(abstract after X)` to the `Numeral Notation`, then numerals larger than `X` are interpreted as the application of the parser to the decimal constant.  Hence we don't have access to the normal form, so cannot perform the translation.

Reply:
Thanks. I think it should be reworded, too.